src/pages/ProjectPage/index.tsx

```
@@ -1,10 +1,13 @@
-import ProjectCards from "@pages/ProjectPage/ProjectCards";
+import ProjectCard from "@pages/ProjectPage/ProjectCard";
+import { Project } from "@_types";
 
 export default function ProjectPage() {
   return (
-    <>
-      <ProjectCards projects={projects} />
-    </>
+    <div className="grid grid-cols-1 md:grid-cols-2 2xl:grid-cols-3 content-center justify-items-center gap-4">
+      {projects.map((project: Project) => (
+        <ProjectCard {...project} />
```

Biome (2.1.2)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)